lib/ladybug/energy_model/gas_equipment_abridged.rb

```diff
@@ -0,0 +1,91 @@
+# this list of conditions and the following disclaimer.
+#
+# (2) Redistributions in binary form must reproduce the above copyright notice,
+# this list of conditions and the following disclaimer in the documentation
+# and/or other materials provided with the distribution.
+#
+# (3) Neither the name of the copyright holder nor the names of any contributors
+# may be used to endorse or promote products derived from this software without
+# specific prior written permission from the respective party.
+#
+# THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDER(S) AND ANY CONTRIBUTORS
+# "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO,
+# THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+# ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER(S), ANY CONTRIBUTORS, THE
+# UNITED STATES GOVERNMENT, OR THE UNITED STATES DEPARTMENT OF ENERGY, NOR ANY OF
+# THEIR EMPLOYEES, BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL,
+# EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT
+# OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+# INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT,
+# STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
+# OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+# *******************************************************************************
+
+require 'ladybug/energy_model/extension'
+require 'ladybug/energy_model/model_object'
+
+module Ladybug
+  module EnergyModel
+    class GasEquipmentAbridged < ModelObject
+      attr_reader :errors, :warnings
+  
+      def initialize(hash = {})
+        super(hash)
+        raise "Incorrect model type '#{@type}'" unless @type == 'GasEquipmentAbridged'
+      end
+    
+      def defaults
+        result = {}
+        result
+      end
+    
+      def find_existing_openstudio_object(openstudio_model)
+        model_gas_equipment = openstudio_model.getGasEquipmentDefinitionByName(@hash[:name])
+        return model_gas_equipment.get unless model_gas_equipment.empty?
+        nil
+      end
+    
+      def create_openstudio_object(openstudio_model)
+        openstudio_gas_equipment_definition = OpenStudio::Model::GasEquipmentDefinition.new(openstudio_model)
+        openstudio_gas_equipment_definition.setWattsperSpaceFloorArea(@hash[:watts_per_area])
+        if @hash[:gas_equipment][:radiant_fraction]
+          openstudio_gas_equipment_definition.setFractionRadiant(@hash[:radiant_fraction])
+        else 
+          openstudio_gas_equipment_definition.setFractionRadiant(@@schema[:definitions][:GasEquipmentAbridged][:properties][:radiant_fraction][:default])
+        end
+        if @hash[:gas_equipment][:latent_fraction]
+          openstudio_electric_equipment_definition.setFractionLatent(@hash[:gas_equipment][:latent_fraction])
```

Comment:
You have a typo here, which is pretty clearly a relic of some copy/pasting.  Maybe this is an indication that both Gas and Electric Equipment should be inheriting from a common class so that you end up with less duplicated code between (this is what I have done on the Python end). Maybe it's a bit much for this PR but you need to at least look through all of the duplicate code for typos, then.